newrelic-agent/build.gradle

```diff
@@ -78,8 +78,6 @@ dependencies {
     shadowIntoJar 'net.sourceforge.jregex:jregex:1.2_01'
 
     shadowIntoJar 'org.apache.httpcomponents:httpclient:4.5.13'
-    // We use this explicitly for its Base64 class. It's also a dependency of httpclient.
-    shadowIntoJar("commons-codec:commons-codec:1.11")
```

Comment:
It doesn't fully remove the dependency on Codec because the Apache HttpClient still requires it. (I actually did just what you suggested, but discovered that it was still being pulled in, so I left 3rd-party notices.)

Reply:
Oh, ok then! LGTM!